sdk/identity/azure-identity/TOKEN_CACHING.md

```diff
@@ -34,6 +34,12 @@ With persistent disk token caching enabled, the library first determines if a va
 - Ensures the app can continue to function during an Azure AD outage or disruption.
 - Avoids having to prompt users to authenticate each time the process is restarted.
 
```

Comment:
Consider: add the missing section on clearing the token cache

Reply:
added a line. Not sure if it needs to be more than that.